integration_test/issue_wizard_test.dart

```diff
@@ -85,10 +85,12 @@ void main() {
         expect(collapsibleFinder, findsOneWidget);
 
         // Expect the question text to be present on this collapsible
-        final questionFinder = find.descendant(
-          of: collapsibleFinder,
-          matching: find.text(question).hitTestable(),
-        );
+        final questionFinder = find
+            .descendant(
+              of: collapsibleFinder,
+              matching: find.text(question),
+            )
+            .hitTestable();
```

Comment:
I don't think this works. I tried it and the same seems to happen.
https://console.firebase.google.com/u/1/project/yivimobile/testlab/histories/bh.c605a71ce4c1b2db/matrices/8866949221523191792/executions/bs.e5aabd3083c1dcb9/test-cases

I think the issue is related to the `scrollUntilVisible`. It seems like the `scrollUntilVisible` thinks that something is on screen and thinks it's finished but it is not `hitTestable` yet.